src/complex_types/inheritance.md

```diff
@@ -76,19 +76,19 @@ namespace DelegatePattern
 endnamespace
 ```
 
-In this design, the EventManager doesn't need to know how the user is notified. It just knows it has a method to do so. We've abstracted the notification mechanism and encapsulated it within specific 'delegate' classes (EmailNotifier, SMSNotifier, PushNotifier). after creating an instance of EventManager, the specific notifier added to the internal array list. This provides a high degree of flexibility.
+In this design, the `EventManager` doesn't need to know how the user is notified—it just knows it has a method to do so. We've abstracted the notification mechanism and encapsulated it within specific "delegate" classes (EmailNotifier, SMSNotifier, PushNotifier) after creating an instance of `EventManager`, the specific notifier added to the internal array list. <!--Jeff, make sure this sentence says what you intended. I removed what appeared to be a rogue period, but I'm not sure the sentence makes sense. Are we saying EventManager is the notifier? Or do we mean "...(EmailNotifier, SMSNotifier, PushNotifier). After creating an instance of EventManager, the specific notifier is added to the internal array list"? Or something else?-->This provides a high degree of flexibility.
```

Comment:
Removing the extra period seems fine, I think the last segment should be "the specific notifier is added to the internal array list."